### PR TITLE
feature(fix-landing-page-redirect):

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
- added a slow down to the page import
- connects and closes unicef/etools-issues#319

### CODE REVIEW

**Comment by daniel-tabarcea** [comment]
connects and closes uncief/etools-issues#319

File: pmp/app-shell.html [hunk 1]
```diff
@@ -317,6 +317,11 @@
         if (!page) {
           return;
         }
+        // sometimes this gets triggered before the element is available, so we need a slow down
+        this.debounce('appShellPageChanged', this._handlePageImport.bind(this, page), 200);
```
Comment (adi130987) [inline pmp/app-shell.html]: I think we should solve this in a different way. `_permissionsPagesStamped`(maybe we need to rename this too) observer should have one more parameter `initialLoad` and will only be triggered when we have `permissions` and when we `initialLoad` becomes `false`(first thing to do in `_lazyLoadPagesAfterCurrentPageHasLoaded` before starting importing the other pages).

Comment (adi130987) [inline pmp/app-shell.html]: We might even not need `_permissionsPagesStamped` at all, and only make the redirect check in `_lazyLoadPagesAfterCurrentPageHasLoaded`

**Comment by adi130987** [comment]
@daniel-tabarcea please review this so we can merge it

**Pull request merged**